app/controllers/org_admin/templates_controller.rb

```diff
@@ -501,18 +501,13 @@ def apply_scoping(scope, customizable = false, all = false)
         
         scopes = calculate_table_scopes(templates, customizations)
 
-        # We scope based on the customizations if the user is NOT a super admin
+        # We scope based on the customizations
         if params[:scope].present? && params[:scope] != 'all'
-          if current_user.can_super_admin?
-            templates = templates.where(published: true) if params[:scope] == 'published'
-            templates = templates.where(published: false) if params[:scope] == 'unpublished'
-          else
-            scoped = templates.select do |t| 
-              c = customizations[t.dmptemplate_id]
-              (params[:scope] == 'unpublished' && (!c.present? || !c.published?)) || (params[:scope] == 'published' && c.present? && c.published?)
-            end
-            templates = Template.where(id: scoped.collect(&:id))
+          scoped = templates.select do |t| 
+            c = customizations[t.dmptemplate_id]
+            (params[:scope] == 'unpublished' && (!c.present? || !c.published?)) || (params[:scope] == 'published' && c.present? && c.published?)
```

Comment:
why do we have this boolean condition here? What are you trying to do? I don't seem to see It is assigned to a variable neither used as part of a conditional construct

Reply:
it determines which templates are selected. If the user clicked on the 'unpublished' filter the params[:scope] would be 'unpublished' so we only want customizations that are unpublished and vice versa when the scope is published.

Reply:
ok, if that's the case, I would refactor params[:scope] outside of that loop. In addition, although more verbose, easier to read 6 months later.
```
if params[:scope] == 'unpublished'
  scoped = templates.select do |t|
    if customizations.has_key?(t.dmptemplate_id)
      !customizations[t.dmptemplate_id].published?
    else
      true
    end
  end
else if params[:scope] == 'published'
  scoped = templates.select do |t|
    customizations.has_key?(t.dmptemplate_id) && customizations[t.dmptemplate_id].published?
  end
else
  scoped = []
end
```